fe-nc-news/src/utils/GetArticle.js

```diff
@@ -1,11 +1,12 @@
-import axios from 'axios'
+import axios from "axios";
 
 const getArticle = (article_id) => {
-  return axios.get(`https://jd-nc-news.onrender.com/api/articles/${article_id}`)
+  return axios
+    .get(`https://jd-nc-news.onrender.com/api/articles/${article_id}`)
     .then((response) => {
       return response.data.article;
     })
     .catch((err) => console.log(err));
```

Comment:
Console.logging an error is not really handling it in a robust enough way. 
You might want to look at catching inside the component (which would also be the end of the promise chain) 
How could you display something to the user in the event an error occured?